airbyte-integrations/bases/base-normalization/normalization/transform_catalog/stream_processor.py

```diff
@@ -701,23 +702,40 @@ def generate_scd_type_2_model(self, from_table: str, column_names: Dict[str, Tup
         if "_ab_cdc_log_pos" in column_names.keys():
             col_cdc_log_pos = self.name_transformer.normalize_column_name("_ab_cdc_log_pos")
             quoted_col_cdc_log_pos = self.name_transformer.normalize_column_name("_ab_cdc_log_pos", in_jinja=True)
-            cdc_updated_order_pattern += f", {col_cdc_log_pos} desc"
+            cdc_updated_order_pattern += f"\n            {col_cdc_log_pos} desc,"
             cdc_cols += f", {cast_begin}{col_cdc_log_pos}{cast_as}" + "{{ dbt_utils.type_string() }}" + f"{cast_end}"
             quoted_cdc_cols += f", {quoted_col_cdc_log_pos}"
 
+        if (
```

Comment:
is this actually reflected in any of the tests? didn't see it in any of the normalization_test_output diffs

Reply:
no, because we don't version every model files for every destinations (it makes a lot of noise in the PRs) and this change only affect bigquery.